airbyte-integrations/connectors/source-stripe/erd/source.dbml

```diff
@@ -1141,6 +1141,8 @@ Table "invoice_line_items" {
     "discounts" array
     "tax_rates" array
     "tax_amounts" array
+    "created" integer
```

Comment:
Nice touch updating the diagrams too! I wonder, should it be timestamp in the diagram?

Reply:
I'm pretty sure I just copied it from one of the other diagrams, so it should be internally consistent (that's not to say that it couldn't be consistently wrong 😄